src/View/JsonApiView.php

```diff
@@ -5,8 +5,8 @@
 use Cake\Core\Configure;
 use Cake\Datasource\RepositoryInterface;
 use Cake\Event\EventManager;
-use Cake\Network\Request;
-use Cake\Network\Response;
+use Cake\Http\Response;
+use Cake\Http\ServerRequest as Request;
```

Comment:
Aliasing it as the old deprecated class name feels like it could lead to misunderstandings to me.